packages/core/admin/server/policies/hasLicenseLimitAllowance.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const utils = require('@strapi/utils');
+const { ApplicationError } = require('@strapi/utils/lib/errors');
+
+const { PolicyError } = utils.errors;
+
+module.exports = async (policyCtx, config = {}) => {
+  if (!strapi.EE) return true;
+
+  const permittedSeats = strapi.ee.licenseInfo.seats;
+  if (!permittedSeats) return true;
+
+  const userCount = await strapi.db.query('admin::user').count({
+    where: { isActive: true },
+  });
+
+  if (userCount < permittedSeats) return true;
+  if (userCount >= permittedSeats && config.isCreating) {
+    throw new PolicyError("License seat limit reached, can't create new user", {
+      policy: 'license-limit-allowance',
+    });
+  }
+
+  const user = await strapi.db.query('admin::user').findOne({
```

Comment:
Just a question, but is there any reason why you're not using the existing user service for this one?

Reply:
No reason to be honest, the only reason is that I didn't look into whether there was a service or no 😅